### PR TITLE
Rough draft

### PR DESCRIPTION
**Pull request by sgharms**
@maxwellbenton 

Here's the update:

## New SWBATs

* Recognize array vocabulary word: Element
* Recognize that arrays are ordered by index
* Recognize `[]` as used for presenting `Array`s
* Recognize that array elements can be accessed through bracket-notation

I think that we can cover push/pop/shift etc in the lab, but I needed these pieces to write this body

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,30 +2,177 @@
...
+
+* Array literal syntax
+* `Array.new`
+
+### Array Literal Syntax
+
+Because showing arrays with brackets is so handy, Ruby added the ability to
+_create_ arrays by simply typing this naturally-readable display content.
+"Literal" here means "readable" like the word "literature" or "literacy." It's
+creating an `Array` by "reading it in."
```
Comment (maxwellbenton) [inline README.md]: I'm not sure if 'reading it in' is clear enough of an explanation. Maybe a narrative would be clearer like - 'you can imagine saying in your head, "I want an array that _literally_ looks like this when displayed."'

File: README.md [hunk 2]
```diff
@@ -2,30 +2,177 @@
...
-* Recap intro
+In this lesson we learned about the data structure `Array`. From iPhone to Java
+to Python, nearly every programming language has something like an `Array`.
+Arrays are composed of _elements_ which are accessed by an _index_. Indexes
+move upward from 0. Arrays are often displayed in programming documentation in
+_array literal syntax_ which is brackets (`[]`) filled with _elements_
+separated by `,`s. Array literal syntax can also be used to create Arrays. A
+lesser-used, but very useful, tool for creating `Array`s is the `Array.new`
+method of Ruby.
+
```
Comment (maxwellbenton) [inline README.md:183]: Maybe add:
```
## Resources

- [Arrays](http://ruby-doc.org/core-2.3.4/Array.html)
- [Literals](https://docs.ruby-lang.org/en/2.0.0/syntax/literals_rdoc.html)
```

File: README.md [hunk 3]
```diff
@@ -2,30 +2,177 @@
...
+create an Array 3 ways:
+
+|documentation|example|result|
+|-------------|-------|------|
+|`new(size=0, default=nil)`| `Array.new(3, "Bark")`|`["Bark", "Bark", "Bark"]`|
+|`new(array)`| `Array.new([1,2,3])`|`[1,2,3]`|
+|`new(size) {\|index\| block }`|`Array.new(3){ \|index\| index ** 2 }`|`[0,2,4]`|
+
+These are, honestly, not that useful day-to-day. Sometimes you might need to
+initialize an `Array` with 12 copies of some name in it like:
```
Comment (maxwellbenton) [inline README.md]: Changed to 16 in second PR for nerd accuracy

Na-na-Na-na-Na-na-Na-na-Na-na-Na-na-Na-na-Na-na-Batman

File: README.md (hunk 2 above)
Comment (sgharms) [reply README.md:183]: @maxwellbenton If a resources section is something to enshrine, and it's documented in https://github.com/learn-co-curriculum/curriculum-standards/blob/master/authoring-checklist.md, you can yell at me about not conforming.

**Comment by sgharms** [comment]
@maxwellbenton Updated after last round of f/b. Are we good to move to the next status?

**Comment by maxwellbenton** [comment]
yep!

**Pull request merged**